core/src/main/java/org/jclouds/Constants.java

```diff
@@ -182,6 +182,11 @@
     */
    public static final String PROPERTY_PROXY_FOR_SOCKETS = "jclouds.proxy-for-sockets";
 
+   /**
+    * List of URI patterns for which the defined proxy will not be used.
+    */
+   public static final String PROPERTY_PROXY_EXCLUDE_URI_PATTERN_PREFIX = "jclouds.proxy-exclude-uri-pattern.";
```

Comment:
What is the format of this?  If it's a list, is it comma-seperated?

Reply:
I was following the PROPERTY_TIMEOUTS_PREFIX example to allow specifying multiple patterns via multiple configuration options. Using commas and such would've required escaping which I preferred to avoid.